src/main/java/de/telran/onlineshopforhomeandgarden1/mapper/OrderMapper.java

```diff
@@ -1,16 +1,56 @@
 package de.telran.onlineshopforhomeandgarden1.mapper;
 
+import de.telran.onlineshopforhomeandgarden1.dto.request.OrderItemRequestDto;
+import de.telran.onlineshopforhomeandgarden1.dto.request.OrderRequestDto;
 import de.telran.onlineshopforhomeandgarden1.dto.response.OrderResponseDto;
 import de.telran.onlineshopforhomeandgarden1.entity.Order;
-import org.mapstruct.Mapper;
+import de.telran.onlineshopforhomeandgarden1.entity.OrderItem;
+import de.telran.onlineshopforhomeandgarden1.entity.Product;
+import de.telran.onlineshopforhomeandgarden1.entity.User;
+import org.mapstruct.*;
 
 
 import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
 
 @Mapper(componentModel = "spring")
 public interface OrderMapper {
```

Comment:
Better to make  OrderMapper abstract class if you need some logic there

Reply:
done